src/vue/modules/buy-back-form/store/index.js

```diff
@@ -50,7 +48,7 @@ export const actions = {
    * @param {String} baseAsset - filter sales by base asset code
    */
   async [types.LOAD_SALE_BY_BASE_ASSET] ({ getters }, baseAsset) {
-    let { data: sales } = await api().get(`${HORIZON_VERSION_PREFIX}/sales`, {
+    let { data: sales } = await api().get(`v3/sales`, {
```

Comment:
Bad thing is to use template interpolating string like \`v3/sales\` without any interpolations
pls make sure that in your changes there is no more template strings with no interpolations